modules/auxiliary/admin/dcerpc/samr_computer.rb

```diff
@@ -187,7 +187,9 @@ def action_add_computer
       user_handle: user_handle,
       user_info: user_info
     )
-    print_good("Successfully created #{@domain_name}\\#{computer_name} with password #{password}")
+    print_good("Successfully created #{@domain_name}\\#{computer_name}")
+    print_good("  Password: #{password}")
+    print_good("  SID:      #{get_computer_sid(computer_name)}")
```

Comment:
Updated this to print the SID automatically when the account is created. The user will probably need it if they intended to use the new account.